content/applications/hr/recruitment.rst

```diff
@@ -26,52 +26,172 @@ is able to be modified to suit the specific recruitment flow for any business.
    The applicant flow with all its stages are universal and applies to all job positions, unless
    specified. :ref:`A specific stage can be configured <recruitment/customize-stages>` to be
    job-specific, meaning that specific stage is only visible for that specific job position.
-   Otherwise, if a new stage is created, or an existing stage is modified, those changes will be
+   Otherwise, if a new stage is created, or an existing stage is modified, those changes are
    visible on all job positions.
 
+.. _recruitment/settings:
+
+Settings
+========
+
+Before creating a job position to be recruited for, the various settings for the *Recruitment* app
+should first be configured. To view and edit the settings, navigate to :menuselection:`Recruitment
+app --> Settings`. After any changes are made, click the :guilabel:`Save` button in the top-left
+corner to save all the changes.
+
+Job posting
+-----------
+
+The *Job Posting* section of the *Recruitment* app settings has only one selection to make. If job
+positions are to be posted to the company's website, enable the :guilabel:`Online Posting` option.
+
+This option only works if the *Website* application is also installed.
+
+Recruitment process
+-------------------
+
+This section of the settings page specifies what the database can and cannot do during the
+recruitment process.
+
+Send interview survey
+~~~~~~~~~~~~~~~~~~~~~
+
+Odoo is capable of having a survey, or exam, sent to an applicant to better determine their skill
+set. This aids the recruitment team and helps them determine which candidates should be moved
+through the recruitment pipeline, and which should no longer be considered.
+
+Enable the :guilabel:`Send Interview Survey` option to be able to send surveys to applicants.
+
+.. note::
+   The *Surveys* application must be installed to use this feature.
+
+Send SMS
+~~~~~~~~
+
+It is possible to send text messages to applicants directly through the *Payroll* app. To do so,
+enable the :guilabel:`Send SMS` option. This option requrtes credits to use. For more information,
```

Comment:
Suggestion to similarly acknowledge the internal link that shows up when the feature is enabled, and to break off the last sentence from the paragraph.

```suggestion
enable the :guilabel:`Send SMS` option. This option requires credits to use, which can be obtained
by clicking the :guilabel:`➡️ Buy credits` internal link that emerges when the feature is enabled.

For more information,
```